cmd/git-sync/main.go

```diff
@@ -59,7 +59,7 @@ var flRoot = flag.String("root", envString("GIT_SYNC_ROOT", envString("HOME", ""
 	"the root directory for git-sync operations, under which --dest will be created")
 var flDest = flag.String("dest", envString("GIT_SYNC_DEST", ""),
 	"the name of (a symlink to) a directory in which to check-out files under --root (defaults to the leaf dir of --repo)")
-var flWait = flag.Float64("wait", envFloat("GIT_SYNC_WAIT", 0),
+var flWait = flag.Float64("wait", envFloat("GIT_SYNC_WAIT", 3),
```

Comment:
This is a pretty significant behavioral change.  Changing it to 1 second I could maybe buy, but 3 feels like a lot.

I am planning to cut a new major version eventually (I have some big, breaking changes I want to make) and we could discuss this in that context.  Can you file an issue so we can keep track of it?

Also, there are at least 2 e2e test cases that do not set --wait, so they will probably break after this change ("webhook" and "http").

Reply:
@thockin I created issue https://github.com/kubernetes/git-sync/issues/244 and reduce the wait time to 1 second. Will look into the test cases.

Reply:
@thockin after updating the default wait time from 3 to 1 second, I'm able to pass all the tests when running `make test` locally.